README.md

```diff
@@ -981,9 +982,52 @@ Optional configuration:
         onFailure(error);
      }
     ```
-## OpenId Connect
 
-Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oidc.html) authentication flow. This flow is required for social authentication, and requires OAuth client registration with Okta. For instructions, see [Social Authentication](http://developer.okta.com/docs/api/resources/social_authentication.html).
+## IdP Discovery
+**:information_source: EA feature:** The Idp Discovery feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).
+
+To enable IdP Discovery into your application, enable Feature Flag `IDP_DISCOVERY` for your Org and configure your Okta admin settings to create discovery policies. Then, set `features.idpDiscovery` to `true` and add additional configs under the `idpDiscovery` key on the [`OktaSignIn`](#new-oktasigninconfig) object.
+
+```javascript
+var signIn = new OktaSignIn({
+  baseUrl: 'https://your-org.okta.com',
+  ... ...
+  idpDiscovery: {
+    requestContext: '/a/app/request/context',
+  },
+  features: {
+    idpDiscovery: true
+  }
+});
+signIn.renderEl(
+  {...},
+  function (res) {
+    if (res.status === 'IDP_DISCOVERY') {
+      res.idpDiscovery.redirectToIdp('/a/app/request/context');
+      return;
+    }
+  }
+);
+```
+
+The IdP Discovery authentication flow will be
+
+1. Display an identifier page to enter Okta userName to discovery IdP for authentication
+2. If IdP is Okta, will transit to Primary Auth
+3. Otherwise is third-party IdP, will invoke the [success callback](#rendereloptions-success-error) with `response.status` as `IDP_DISCOVERY`.
```

Comment:
"If the IdP is a 3rd party IdP or a different Okta org, the widget will invoke the ...."